nuxeo-services/nuxeo-platform-web-common/src/main/java/org/nuxeo/ecm/platform/ui/web/auth/NuxeoAuthenticationFilter.java

```diff
@@ -449,6 +449,9 @@ public void doFilterInternal(ServletRequest request, ServletResponse response, F
 
         NuxeoAuthenticationPropagator.CleanupCallback propagatedAuthCb = null;
 
+        String forceAnonymousLoginParam = httpRequest.getParameter(FORCE_ANONYMOUS_LOGIN);
+        boolean forceAnonymousLogin = "true".equals(forceAnonymousLoginParam);
```

Comment:
We usually even use `Boolean.parseBoolean(forceAnonymousLoginParam)` in these situations.